src/components/Header/Header.jsx

```diff
@@ -2,111 +2,154 @@ import { Badge, Button, Flex } from "@radix-ui/themes";
-                        <Button size="4" variant="ghost" color="green">
-                            Mis gimnasios
-                        </Button>
-                    </Link>
-                    <Link to="/owner/equipments">
-                        <Button size="4" variant="ghost" color="green">
-                            Mis máquinas
-                        </Button>
-                    </Link>
-                    <Link to="/owner/users">
-                        <Button size="4" variant="ghost" color="green">
-                            Usuarios
-                        </Button>
-                    </Link>
-                    <Link to="/owner/tickets">
-                        <Button size="4" variant="ghost" color="green">
-                            Tickets
-                        </Button>
-                    </Link>
-                    </>
-                ) : user?.rol === "client" ? (
-                    <>
-                    <Link to="/user/routines">
-                        <Button size="4" variant="ghost" color="green">
-                            Rutinas
-                        </Button>
-                    </Link>
-                    <Link to="/user/add-tickets">
-                        <Button size="4" variant="ghost" color="green">
-                            Crear ticket
-                        </Button>
-                    </Link>
-                    </>
-                ) : null}      
-            </div>
-            </div>
-
-                
-            
-            <div className="flex flex-col md:flex-row gap-3 mb-5 md:mb-0 md:mr-4 items-center">
-                {user ? (
-                    <>
-                    <Link to="/profile">
-                        <Button size="3" variant="surface" color="green">
-                            {user.username}
-                        </Button>
-                    </Link>
-                    <Link to="/">
-                        <Button size="3" variant="solid" color="green" onClick={logoutUser}>
-                            Salir
-                        </Button>
-                    </Link>
-                    </>
-                ) : (
-                    <>
-                    <Link to="/login">
-                            <Button size="2" variant="solid" color="green">
-                                Entrar
-                            </Button>
-                        </Link><Link to="/register-client">
-                                <Button size="2" variant="surface" color="green">
-                                    Registrarse
-                                </Button>
-                            </Link>
-                            </>
-                ) }
-
-                
-                
-                
-            </div>
-        </header>
+          )}
+          <div className="flex ml-auto lg:order-1 gap-4 sm:mb-0 mb-4">
+            {user ? (
+              <>
+                <Link to="/profile">
```

Comment:
```suggestion
                <Link to="/profile">
```
Estas poniendo que va a http://localhost:5173/profile y eso no existe.
**Cuidado con las rutas absolutas** que te cargas toda la url al ponerle una barra como prefijo.
Por eso yo pongo rutas relativas sin la barra por delante

Reply:
Se que no existe, pero es a la ruta a la que tiene que ser dirigida cuando esté implementado correctamente, por eso está puesto de esa manera